tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27, py36, py38
```

Comment:
I found two other things and all other looks ok.
Unfortunately I can't comment on unchanged line so:
1) We mix two approach:
We have requirements.txt file and install_requires in setup.py and their are not connected
2) We should add support for multiple celery versions. Maybe we can use build matrix.

The second one looks more connected with this PR. Maybe we can create another task for it.

Reply:
Because we rely more on celery version that on python version.

Reply:
Feel free to fix that in other PR :)